### PR TITLE
rework how the installation directory is specified for data files

### PR DESCRIPTION
**Pull request by ayan-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

**Comment by ayan-usgs** [comment]
This should remove the need for a symlink.

File: wdfn-server/setup.py [hunk 1]
```diff
@@ -61,26 +67,33 @@ def identify_data_files(directory_names, exclusions=('.gitignore', '.webassets-c
...
+    test_suite='unittest:TestLoader',
+    zip_safe=False,
+    # include the tier agnostic configuration file in the distributable
+    # the file gets placed in site-packages upon dist installation
+    py_modules=['config'],
+    # include static files in the distributable
+    # they will appear in the root of the virtualenv upon dist installation
+    data_files=identify_data_files(
+        [
+            ('assets', '../assets/dist'),
```
Comment (danielnaab) [inline wdfn-server/setup.py]: Would this also require a change in the static symlink from './assets/dist` to `./assets`?

Comment (ayan-usgs) [reply wdfn-server/setup.py]: I'm not sure I follow. Where is that `./assets/dist` symlink?

Comment (danielnaab) [reply wdfn-server/setup.py]: This one: https://code.usgs.gov/WDFN/waterdataui_config/blob/master/roles/application/tasks/deploy.yml#L50-54

**Pull request merged**